tests/components/utility_meter/test_sensor.py

```diff
@@ -1637,8 +1638,17 @@ async def _test_self_reset(
 
     now += timedelta(seconds=30)
     with freeze_time(now):
+        # Listen for events and check that state in the first event after reset is actually 0, issue #142053
+        events = []
+        unsub = async_track_state_change_event(
```

Comment:
```suggestion
        async def handle_energy_bill_event(event):
    events.append(event)

unsub = async_track_state_change_event(
    hass,
    "sensor.energy_bill",
    handle_energy_bill_event,
)
```

Be sure to import `from homeassistant.helpers.event import async_track_state_change_event` at the top and I believe this should clear your `# pylint: disable-next=unnecessary-lambda`. :)

Reply:
It is imported, but it's still warning. It's the same in other tests.
I choose this form because it's shorter :-) I committed this suggestion.